assets/js/stripe-payment-request.js

```diff
@@ -350,8 +361,8 @@ jQuery( function( $ ) {
 					country: cart.order_data.country_code,
```

Comment:
Thanks for pointing out this. I agree this needs some improvement. How about we address this in a separate PR as well as changing what I mentioned on my [comment](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1577/files#r645288392) above: how the PR button is initialized on the Product page since it will definitely affect these lines.

Reply:
My initial idea was just about removing one of these variables and then adapting below where the other variable is used, but if you have something more complex in mind along with refactoring how the button is initialized on the product page, it's ok to do it in a separate PR.

Reply:
See my comment https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1577#discussion_r645782332